test_communication/test/message_fixtures.py

```diff
@@ -98,17 +98,18 @@ def get_msg_primitives():
     msg.bool_value = True
     msg.byte_value = bytes([1])
     msg.char_value = '\x01'
-    msg.float32_value = float(1.125)
-    msg.float64_value = float(1.125)
-    msg.int8_value = int(1)
-    msg.uint8_value = int(1)
-    msg.int16_value = int(1)
-    msg.uint16_value = int(1)
-    msg.int32_value = int(1)
-    msg.uint32_value = int(1)
-    msg.int64_value = int(1)
-    msg.uint64_value = int(1)
+    msg.float32_value = float(1.0)
+    msg.float64_value = float(1.0)
```

Comment:
actually no c++ is using 1.0 and only in this specific message (c.f. https://github.com/ros2/system_tests/blob/master/test_communication/test/message_fixtures.hpp#L106) that's why I changed python accordingly. I hesitated which one to change and given that this message seemed to aim to send 1 in every field I modified the python one.

I don't have srtong feeling about it and change both of them for 1.125 if it makes more sense to everyone